artifacts/mysql-statefulset.yaml

```diff
@@ -128,24 +137,24 @@ spec:
           # Check if we need to complete a clone by starting replication.
           if [[ -f change_master_to.sql.in ]]; then
             echo "Waiting for mysqld to be ready (accepting connections)"
-            until mysql -h 127.0.0.1 -e "SELECT 1"; do sleep 1; done
+            until mysql -h 127.0.0.1 -p${MYSQL_ROOT_PASSWORD} -e "SELECT 1"; do sleep 1; done
 
             echo "Initializing replication from clone position"
             # In case of container restart, attempt this at-most-once.
             mv change_master_to.sql.in change_master_to.sql.orig
-            mysql -h 127.0.0.1 <<EOF
+            mysql -h 127.0.0.1 -p${MYSQL_ROOT_PASSWORD} <<EOF
           $(<change_master_to.sql.orig),
             MASTER_HOST='{{ .Name }}-0.{{ .Name }}',
             MASTER_USER='root',
-            MASTER_PASSWORD='',
+            MASTER_PASSWORD='${MYSQL_ROOT_PASSWORD}',
             MASTER_CONNECT_RETRY=10;
           START SLAVE;
           EOF
           fi
 
           # Start a server to send backups when requested by peers.
           exec ncat --listen --keep-open --send-only --max-conns=1 3307 -c \
-            "xtrabackup --backup --slave-info --stream=xbstream --host=127.0.0.1 --user=root"
+            "xtrabackup --backup --slave-info --stream=xbstream --host=127.0.0.1 --user=root -p${MYSQL_ROOT_PASSWORD}"
```

Comment:
Maybe `--password=${MYSQL_ROOT_PASSWORD}`, to keep consistent flags usage.

Reply:
Done.